napari/utils/events/containers/_nested_list.py

```diff
@@ -465,8 +414,3 @@ def _iter_indices(self, start=0, stop=None, root=(), deep=False):
                         yield i
         else:
             yield from super()._iter_indices(start, stop)
-
-
-def _le(y):
-    # create a new function that accepts a single value x, returns x < y
-    return lambda x: x <= y
```

Comment:
Ah sorry I didn't saw that _le was a move a the function previously defined here.

Reply:
nonetheless... I replaced with list comp, reads way better.